src/panel/Menu.jsx

```diff
@@ -21,9 +20,11 @@ const Menu = () => {
     };
   }, []);
 
-  const open = () => {
-    Telemetry.add(Telemetry.MENU_CLICK);
-    setIsOpen(true);
+  const toggle = () => {
+    if (!isOpen) {
+      Telemetry.add(Telemetry.MENU_CLICK);
```

Comment:
Ok, so we can expect 2 times less registered events from now IIUC

Reply:
Yep, and nobody cares :)